pkg/receive/multitsdb_test.go

```diff
@@ -425,7 +425,7 @@ func TestMultiTSDBPrune(t *testing.T) {
 			name:            "prune tsdbs with object storage",
 			bucket:          objstore.NewInMemBucket(),
 			expectedTenants: 2,
-			expectedUploads: 1,
+			expectedUploads: 2,
```

Comment:
This is now `2` because background syncs are constantly running, so the compacted tenant gets uploaded.